app/frontend/components/pages/delegations/delegatePage.tsx

```diff
@@ -46,58 +49,148 @@ const BigDelegatorOffer = (): h.JSX.Element => (
+    const pool =
+      validStakepoolDataProvider.getPoolInfoByPoolHash(fieldValue) ||
+      validStakepoolDataProvider.getPoolInfoByTicker(fieldValue)
+    if (pool) return {poolHash: pool.poolHash, error: null}
+
+    const hasTickerMapping = validStakepoolDataProvider.hasTickerMapping
+    const isTickerString = fieldValue.length <= 5 && fieldValue.toUpperCase() === fieldValue
+    const poolHash = null
+    if (!hasTickerMapping && isTickerString) {
+      return {poolHash, error: {code: 'TickerSearchDisabled'}}
+    }
+    return {poolHash, error: {code: 'InvalidStakepoolIdentifier', params: {hasTickerMapping}}}
+  }
+
+  const pool = validStakepoolDataProvider.getPoolInfoByPoolHash(fieldValue)
+  if (pool) return {poolHash: pool.poolHash, error: null}
+  return {
+    poolHash: null,
+    error: {code: 'InvalidStakepoolIdentifier', params: {hasTickerMapping: false}},
+  }
+}
+
 interface Props {
-  stakePool: Stakepool
-  delegationFee: Lovelace
-  calculatingDelegationFee: boolean
-  delegationValidationError: any
-  isShelleyCompatible: boolean
-  txSuccessTab: string
-  gettingPoolInfo: boolean
-  poolRecommendation: PoolRecommendation
-  pool: Stakepool
-  isBigDelegator: boolean
   withAccordion: boolean
   title: string
-  confirmTransaction: (txConfirmType) => Promise<void>
-  selectAdaliteStakepool: () => void
 }
 
-const Delegate = ({
-  stakePool,
-  delegationFee,
-  calculatingDelegationFee,
-  delegationValidationError,
-  isShelleyCompatible,
-  txSuccessTab,
-  gettingPoolInfo,
-  poolRecommendation,
-  pool,
-  isBigDelegator,
-  withAccordion,
-  title,
-  confirmTransaction,
-}: Props): h.JSX.Element => {
-  /*
-  REFACTOR (calculateFee):
-  Calculation of "fee" should be fired from there explicitely using "useEffect"
-  when component mounts.
-  */
+const Delegate = ({withAccordion, title}: Props): h.JSX.Element => {
+  const {
+    txSuccessTab,
+    stakePool,
+    currentDelegation,
+    calculatingDelegationFee,
+    delegationFee,
+    delegationValidationError,
+    gettingPoolInfo,
+    isBigDelegator,
+    isShelleyCompatible,
+    poolRecommendation,
+    validStakepoolDataProvider,
+  } = useSelector((state) => ({
+    // REFACTOR: (Untyped errors)
+    delegationValidationError: state.delegationValidationError,
+    stakePool: state.shelleyDelegation.selectedPool,
+    currentDelegation: getSourceAccountInfo(state).shelleyAccountInfo.delegation,
+    calculatingDelegationFee: state.calculatingDelegationFee,
+    delegationFee: state.shelleyDelegation.delegationFee,
+    txSuccessTab: state.txSuccessTab,
+    gettingPoolInfo: state.gettingPoolInfo,
+    isShelleyCompatible: state.isShelleyCompatible,
+    poolRecommendation: getSourceAccountInfo(state).poolRecommendation,
+    isBigDelegator: isBigDelegatorSelector(state),
+    validStakepoolDataProvider: state.validStakepoolDataProvider,
+  }))
+  const {confirmTransaction, updateStakePoolIdentifier} = useActions(actions)
```

Comment:
Seems like `useActions` returns new reference of `updateStakePoolIdentifier` :thinking: